newsfragments/3408.breaking.rst

```diff
@@ -0,0 +1,5 @@
+Upgrades to use latest ``ABI`` utilities and typings from ``eth-utils`` and ``eth-typing``.
+* New ``ABI`` functions from ``eth-utils`` are now included in the ``web3.utils.abi`` module.
```

Comment:
It was included here because we decided the `eth-utils` upgrade itself was breaking. I can leave it out of this though since we have it covered in the other newsfragment.

Reply:
We can either remove or rephrase it as something that is actually breaking like:

```suggestion
* New versions of existing ABI functions were added to ``eth-utils`` and are now exposed in `web3.py` via ``web3.utils.abi``.
```